graphql_server/__init__.py

```diff
@@ -293,7 +293,7 @@ def get_response(
         if validation_errors:
             return ExecutionResult(data=None, errors=validation_errors)
 
-        execution_result = execute(
+        execution_result = experimental_execute_incrementally(
```

Comment:
We might want to expose this in a different method (`get_response_incrementally`) so we can feature flag this instead of automatically using it when we update our requirement.